Alexey/Files/mp3_tagger/main/mp3_tagger.py

```diff
@@ -0,0 +1,51 @@
+import eyed3
+
+
+def main():
+    file_name = str(input('Enter file name: ')) + '.mp3'
+    answer = str(input('Enter what you want to do: '))
+    audio = eyed3.load(file_name)
+    if answer == 'edit':
```

Comment:
provide 'help' option to get available commands

Reply:
done